pypeit/find_objects.py

```diff
@@ -481,7 +482,9 @@ def get_platescale(self, slitord_id=None):
 
 
     def global_skysub(self, skymask=None, update_crmask=True, trim_edg = (0, 0),
-                      previous_sky=None, show_fit=False, show=False, show_objs=False, objs_not_masked=False):
+                      previous_sky=None, show_fit=False, show=False, 
```

Comment:
`global_skysub` is also defined for the child class for IFU. You should add `reinit_bpm` there as well.